### PR TITLE
fix: snippet syntax, add the use statement

### PR DESCRIPTION
**Pull request by beauwilliams**

### CODE REVIEW

**Comment by beauwilliams** [comment]
I had an issue after copy pasting the snippet so I fixed it up

File: README.md [hunk 1]
```diff
@@ -29,25 +29,27 @@ General framework for context aware hover providers (similar to `vim.lsp.buf.hov
...
+                require("hover.providers.dictionary")
+            end,
+            preview_opts = {
+                border = "rounded"
+            },
+            title = true
+        }
+
+        -- setup keymaps
+        vim.keymap.set("n", "gk", require("hover").hover_select, {desc = "hover.nvim (select)"})
```
Comment (lewis6991) [inline README.md:50]: This is missing the other keymap

File: README.md [hunk 2]
```diff
@@ -29,25 +29,27 @@ General framework for context aware hover providers (similar to `vim.lsp.buf.hov
...
+        require("hover").setup {
+            init = function()
+                -- require providers
+                require("hover.providers.lsp")
+                -- require('hover.providers.gh')
+                require("hover.providers.man")
+                require("hover.providers.dictionary")
+            end,
+            preview_opts = {
+                border = "rounded"
```
Comment (vladdoster) [inline README.md:44]: @lewis6991 This should probably remain `nil` aswell

File: README.md [hunk 3]
```diff
@@ -29,25 +29,27 @@ General framework for context aware hover providers (similar to `vim.lsp.buf.hov
...
+use ({
+    "lewis6991/hover.nvim",
+    config = function()
+        require("hover").setup {
+            init = function()
+                -- require providers
+                require("hover.providers.lsp")
+                -- require('hover.providers.gh')
+                require("hover.providers.man")
+                require("hover.providers.dictionary")
```
Comment (vladdoster) [inline README.md:41]: These should be commented out

**Comment by beauwilliams** [comment]
Sorry about that guys. Had a stupid moment and realised now I made a mistake. Fixed up now.

**Comment by williamboman** [comment]
<details><summary>Might I suggest the following syntax and indentation instead (no wrapping parantheses):</summary>

```lua
use {
    "lewis6991/hover.nvim",
    config = function()
        require("hover").setup {
            init = function()
                -- Require providers
                require("hover.providers.lsp")
                -- require('hover.providers.gh')
                -- require('hover.providers.man')
                -- require('hover.providers.dictionary')
            end,
            preview_opts = {
                border = nil
            },
            title = true
        }

        -- Setup keymaps
        vim.keymap.set("n", "K", require("hover").hover, {desc = "hover.nvim"})
        vim.keymap.set("n", "gK", require("hover").hover_select, {desc = "hover.nvim (select)"})
    end
}
```

</details>

**Comment by beauwilliams** [comment]
Agreed, done!

**Pull request merged**